docs/code_of_conduct.md

```diff
@@ -20,13 +20,22 @@ Private or public harassment of any kind will not be tolerated. Since harassment
 
 Additionally, spam and other content which disrupts or prevents LDK contributors from working on LDK is not acceptable.
 
+In case of a concrete issue that raises a deadlock about the application of the code of conduct, 2 additional members should be selected by consensus of the code of conduct team from the community of
+active LDK contributors to join the team in its decision-making process to adjudicate the case. The decision of the majority of the 5 members is binding.
+
```

Comment:
Does the "normal" CoC always consist of 3 members? What would an example of a deadlock look like here?

Reply:
Yes “normal” CoC currently consists of 3 members. Note latest modification proposes to bump the number to 5 at next nomination in 2025 (for the staggered approach).

Latest changes introduces the definition of a deadlock, which is just when there is no unanimity between the 3 CoC members on the decision to take and therefore bumping to 5 allow to unlock the situation, hopefully. So in practice a deadlock can be any material event defined as unacceptable behavior, it’s just when there is no formal unanimity in the decision-making process.